### PR TITLE
[1LP][RFR] Added test_storage_manager_navigation_from_cloudprovider

### PR DESCRIPTION
**Pull request by mmojzis**
Added test_storage_manager_navigation_from_cloudprovider automated test and caseimportance.
It should whether Storage Manager is accessible from Cloud Provider
@digitronik Please review

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 597bdd866c00ccd6824e57acb149a2f1c8f9c979:

`cfme/tests/storage/test_manager.py`:
- :heavy_exclamation_mark: Line 141:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/storage/test_manager.py [hunk 1]
```diff
@@ -97,10 +99,42 @@ def test_storage_manager_delete(manager, provider_cleanup):
...
+        assignee: mmojzis
+        initialEstimate: 1/4h
+        casecomponent: Cloud
+        caseimportance: high
+    """
+    view = navigate_to(provider, 'Details')
+    manager_count = int(view.entities.summary("Relationships").get_text_of("Storage Managers"))
+    assert manager_count > 0
+    view.entities.summary("Relationships").click_at("Storage Managers")
+    storage_view = view.browser.create_view(ProviderStorageManagerAllView)
```
Comment (john-dupuy) [inline cfme/tests/storage/test_manager.py]: Question:  Why not make a `storage_manager` object and then navigate via that object? Will the navigation be different in that case? (i.e. won't click the `storage_manager` from the provider summary page)

File: cfme/tests/storage/test_manager.py [hunk 2]
```diff
@@ -97,10 +99,42 @@ def test_storage_manager_delete(manager, provider_cleanup):
...
     assert not manager.exists
+
+
+def test_storage_manager_navigation_from_cloudprovider(manager, provider):
+    """ Test whether Storage Manager is accessible from Cloud Provider
+
+    prerequisites:
+        * Storage provider
+
+    Steps:
```
Comment (lina-nikiforova) [inline cfme/tests/storage/test_manager.py]: The steps should be under Polarion tag

**Comment by lina-nikiforova** [comment]
@mmojzis can you check why this test was skipped for `5.10z`?
`cfme/tests/storage/test_manager.py::test_storage_manager_navigation_from_cloudprovider[block_manager-ec2] SKIPPED`

File: cfme/tests/storage/test_manager.py (hunk 2 above)
Comment (jawatts) [reply cfme/tests/storage/test_manager.py]: under 'testSteps' key

File: cfme/tests/storage/test_manager.py (hunk 1 above)
Comment (jawatts) [reply cfme/tests/storage/test_manager.py]: He is testing that you can navigate to the Storage Managers page via the Cloud Providers by click the "Relationships" table. This is going to be a different navigate than just navigating to the storage_manager object directly.

Comment (mmojzis) [reply cfme/tests/storage/test_manager.py]: Yes, that's what I do there.

File: cfme/tests/storage/test_manager.py (hunk 2 above)
Comment (mmojzis) [reply cfme/tests/storage/test_manager.py]: Solved.

**Comment by mmojzis** [comment]
> @mmojzis can you check why this test was skipped for `5.10z`?
> `cfme/tests/storage/test_manager.py::test_storage_manager_navigation_from_cloudprovider[block_manager-ec2] SKIPPED`

I checked it. Reran the tests. Now 5.10 is okay and 5.9 is failing on sprout. 
In my env it runs correctly.

**Pull request merged**